src/Chainweb/Chainweb/Configuration.hs

```diff
@@ -555,4 +559,40 @@ pChainwebConfiguration = id
         % long "module-cache-limit"
         <> help "Maximum size of the per-chain checkpointer module cache in bytes"
         <> metavar "INT"
+    where
+    knownVersion = do
+        option (findKnownVersion =<< textReader)
+            % long "chainweb-version"
+            <> short 'v'
+            <> help "the chainweb version that this node is using"
+    version = constructVersion
```

Comment:
Consider using the `ConfigValidation` and define the command line parse for Version as separate function.